vpr/src/route/router_lookahead_map_utils.cpp

```diff
@@ -1457,9 +1454,11 @@ static void adjust_rr_pin_position(const RRNodeId rr, int& x, int& y) {
         y -= 1;
         y = std::max(y, 0);
     }
+
+    return {x, y};
 }
 
-static void adjust_rr_wire_position(const RRNodeId rr, int& x, int& y) {
+static std::pair<int, int> adjust_rr_wire_position(const RRNodeId rr) {
```

Comment:
Should comment the .h file saying it gives the drive point of the wire (if it directional) or its middle if it is bidirectional.